samples/iterable_blob_dataset_sample.py

```diff
@@ -0,0 +1,87 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: iterable_blob_dataset_sample.py
```

Comment:
Instead of partitioning the dataset samples by classes, it probably makes sense to do it by class methods (e.g. `from_container_url` and `from_blob_urls`) mainly when customers come to find samples they are more likely to be searching by class method since the two differ fairly significantly in both call pattern and use case. So that will make it easier to find based on the use case a customer is looking for.